components/server/src/github/github-auth-provider.ts

```diff
@@ -82,26 +82,40 @@ export class GitHubAuthProvider extends GenericAuthProvider {
         }
 
         const fetchCurrentOrg = async () => {
-            if(!org) return {state:"active"};
-            const response = await api.orgs.getMembershipForAuthenticatedUser({
-                org,
-            });
-            if (response.status !== 200) {
-                throw new GitHubApiError(response);
+            log.info(`fetchCurrentOrg: ${org}`);
+
+            if (!org) return {state:"active"};
+
+            if (org.includes(',')) {
+                log.info('multiple orgs');
             }
-            const {state} = response.data;
-            if(state != "active"){
-                throw new GitHubApiError(response);
+
+            const orgs = org.split(',').map(x => x.trim());
+
+            for (const orgName of orgs) {
+                log.info(`fetchCurrentOrg org: ${orgName}`);
+                try {
+                    const response = await api.orgs.getMembershipForAuthenticatedUser({ org: orgName });
+                    log.info(`Status: ${response.status}, State: ${response.data.state}`);
+                    if (response.status === 200 && response.data.state === 'active') {
+                        return response.data.state;
```

Comment:
this should be {state:"active"}; ?  otherwise we are just returning string

Reply:
Hi @pavan-tri 
Original code:
![image](https://user-images.githubusercontent.com/7335786/134855873-62ebd4b6-8ea3-4360-92f7-165997f449d8.png)